package.json

```diff
@@ -1,13 +1,15 @@
 {
-  "name": "@itwin/reality-capture-main",
+  "name": "@itwin/reality-capture",
```

Comment:
prob want to add private field to this package.json so it doesnt try to publish this